demo/main.js

```diff
@@ -0,0 +1,140 @@
+document.body.scrollTop += 1;
```

Comment:
Browser compatibility crap. Some browsers use `document.body.scrollTop` others use `document.documentElement`. For those that don't use `document.body.scrollTop`, incrementing it will leave it at 0. Hence: increment `document.body.scrollTop`, check if its != 0, then set the right one.

Reply:
Oh interesting, does `window.scrollY` not work help here?